mcpjam-inspector/client/src/components/clients/redesigned/canvas/sandbox-config-grid.css

```
@@ -0,0 +1,121 @@
+  align-items: baseline;
+  gap: 4px 8px;
+  min-width: 0;
+}
+.sandbox-config-grid .hp-sb-value {
+  text-align: left;
+  font-family: ui-monospace, "JetBrains Mono", monospace;
+  font-size: 11.5px;
+  color: var(--hp-sandbox-ink);
+  font-weight: 500;
+  min-width: 0;
+  word-break: break-word;
```

Stylelint (17.11.1)

[error] Deprecated keyword "break-word" for property "word-break" (declaration-property-value-keyword-no-deprecated)

(declaration-property-value-keyword-no-deprecated)